app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -62,47 +63,47 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    const String projectId = 'flutter-dashboard';
-    const String dataset = 'cocoon';
-    const String table = 'Checklist';
-
+    final GithubService githubService = await config.createGithubService();
     final GitHub github = await config.createGitHubClient();
     const RepositorySlug slug = RepositorySlug('flutter', 'flutter');
-    final Stream<RepositoryCommit> commits =
-        github.repositories.listCommits(slug);
+    final Stream<Branch> branches = github.repositories.listBranches(slug);
     final DatastoreService datastore = datastoreProvider();
-    final TabledataResourceApi tabledataResourceApi =
-        await config.createTabledataResourceApi();
-    final List<Commit> newCommits = <Commit>[];
-    final List<Map<String, Object>> tableDataInsertAllRequestRows =
-        <Map<String, Object>>[];
+    final RegExp exp = RegExp(r'v[0-9]+.[0-9]+.[0-9]+');
```

Comment:
No need to explicitly exclude the existing hotfix branches - just change the regexp to be stricter - r`^rv[0-9]+\.[0-9]+\.[0-9]+$`

Reply:
Thanks for the suggestion.